bot/exts/core/error_handler.py

```diff
@@ -64,17 +68,27 @@ async def on_command_error(self, ctx: Context, error: CommandError) -> None:
             embed = self._get_error_embed("Permission error", "You are not allowed to use this command!")
             await ctx.send(embed=embed)
             return
+        if isinstance(error, InMonthCheckFailure):
+            embed = self._get_error_embed("Command not available", str(error))
+            await ctx.send(embed=embed)
+            return
+        if isinstance(error, InWhitelistCheckFailure):
```

Comment:
We could do with adding a handler for CheckFailure too, so that we don't get people spamming this error by running one of the admin-only commands.
Doesn't need to be in this PR though.
![image](https://github.com/python-discord/sir-robin/assets/9753350/1c0c9666-75f6-43c1-9958-53947237d8e4)